cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaReconciler.java

```diff
@@ -181,11 +186,13 @@ public KafkaReconciler(
             ClusterOperatorConfig config,
             ResourceOperatorSupplier supplier,
             PlatformFeaturesAvailability pfa,
-            Vertx vertx
+            Vertx vertx,
+            KubernetesRestartEventPublisher eventsPublisher
```

Comment:
Why is not taken from the Resource Supplier?

Reply:
:+1: